studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -128,11 +136,21 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
                     </Tooltip.Portal>
                   )}
                 </Tooltip.Root>
-                <Link href={`/project/${ref}/settings/general`}>
-                  <a>
-                    <Button type="default">View project settings</Button>
-                  </a>
-                </Link>
+                {isFreePlan ? (
+                  <Link
+                    href={`/project/${ref}/settings/billing/subscription?panel=subscriptionPlan`}
```

Comment:
Let's have the correct URL depending on org/project level billing. We've set up some redirects, but it's still a longer route to go through the project first and then be redirected to the org. So a better UX to get the URL right

Org-Billing URL:

`/org/${slug}/billing?panel=subscriptionPlan`

To check if the org uses org level billing

```ts
 const billedViaOrg = Boolean(currentOrg?.subscription_id)
```